samples/Blazor/BlazorServer/Services/TodoService.cs

```diff
@@ -69,33 +68,33 @@ await _client.GetGrain<ITodoManagerGrain>(ownerKey)
     public Task<StreamSubscriptionHandle<TodoNotification>> SubscribeAsync(
         Guid ownerKey, Func<TodoNotification, Task> action) =>
         _client.GetStreamProvider("MemoryStreams")
-            .GetStream<TodoNotification>(ownerKey, nameof(ITodoGrain))
+            .GetStream<TodoNotification>(ownerKey)
             .SubscribeAsync(new TodoItemObserver(_logger, action));
+}
 
-    private class TodoItemObserver : IAsyncObserver<TodoNotification>
-    {
-        private readonly ILogger<TodoService> _logger;
-        private readonly Func<TodoNotification, Task> _onNext;
-
-        public TodoItemObserver(
-            ILogger<TodoService> logger,
-            Func<TodoNotification, Task> action)
-        {
-            _logger = logger;
-            _onNext = action;
-        }
+sealed file class TodoItemObserver : IAsyncObserver<TodoNotification>
```

Comment:
"file" is meant primarily for Source Generators, AFAIK

Reply:
It limits the scope of the type to only within the file. Since the `TodoItemObserver` was only used within this file, it felt appropriate, no?

Reply:
Yeah, it's fine, I have no problems with it.